lib/dotcom_web/controllers/page_controller.ex

```diff
@@ -23,6 +23,7 @@ defmodule DotcomWeb.PageController do
   def index(conn, _params) do
     {promoted, remainder} = whats_happening_items()
     banner = banner()
+    photo = Repo.photo()
```

Comment:
For both of these, unless we're actually doing something with the variable, they can just be inlined in the assigns.

```
import CMS.Repo, only: [photo: 0]

|> async_assign_default(:photo, &photo/0)
```